build/git-history.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const path = require("path");
+
+const { CONTENT_ROOT, execGit } = require("../content");
+
+function getFromGit(contentRoot = CONTENT_ROOT) {
+  const repoRoot = execGit(["rev-parse", "--show-toplevel"], {
+    cwd: contentRoot,
+  });
+
+  const MARKER = "COMMIT:";
+  const output = execGit(
+    [
+      "log",
+      "--name-only",
+      "--no-decorate",
+      `--format=${MARKER}%cI`,
+      "--date-order",
+      "--reverse",
+    ],
+    {
+      cwd: repoRoot,
+    }
+  );
+
+  const map = new Map();
+  let date = null;
+  for (let line of output.split("\n")) {
+    // Happens to file paths that contain non-ascii or control charaters.
```

Comment:
a solution to this would be to add `-z` to the git command and then split by `"\0"`. That way git does not escape non-ascii symbols.